src/validators/config.rs

```diff
@@ -0,0 +1,49 @@
+use std::borrow::Cow;
+use std::str::FromStr;
+
+use base64::Engine;
+use pyo3::types::{PyDict, PyString};
+use pyo3::{intern, prelude::*};
+
+use crate::errors::ErrorType;
+use crate::input::EitherBytes;
+use crate::serializers::BytesMode;
+use crate::tools::SchemaDict;
+
+#[derive(Default, Debug, Clone, Copy, PartialEq, Eq)]
+pub struct ValBytesMode {
+    pub ser: BytesMode,
+}
```

Comment:
Is there a reason this needed to be a separate type from `BytesMode`?

(Maybe rather than coupling the `validators` code to the `serializers` code, does it make sense to make a new `common` shared location?)

Reply:
`ValBytesMode::from_config` parses a different config key than `BytesMode::from_config`. I'm not sure how to modify the various `from_config` usages if these became one type.

It also seems to be a bit tricky to move serialization config to a `common` package because it depends on `serializers::errors::py_err_se_err`, but this seems like a secondary problem that could be fixed.

I think it probably makes sense for these to be separate types in their respective packages, but I'm also happy to make changes—let me know what you prefer.

Reply:
Right I see. That's a bigger refactor for another time, so let's keep the implementation as-is for now.